app/common/lib/shieldUtil.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const frameStateUtil = require('../../../js/state/frameStateUtil')
```

Comment:
I think we would like to avoid adding new *Util classes. Maybe add this to `frameState.js` and pass in state and frameKey?